### DISCUSSION TITLE
MariaDB instalation

### DISCUSSION THREAD

**Discussion by kotovasia5120** [tteck/Proxmox#1804] [Q&A] [answered]
Hello. I don't understand what these lines in the instructions mean.
> Change the recorder: db_url: in your HA configuration.yaml
> 
> Example:
> 
> recorder:
>   db_url: mysql://admin:password@192.168.100.26:3306/homeassistant?charset=utf8mb4

**Comment by Vakrehus** [comment, marked as answer]
This has nothing to do with Proxmox and is Home Assistant specific.

Уверен, что Путин сможет ответить на этот вопрос.
